storage/innobase/xtrabackup/test/suites/keyring/innodb_keyring_file_component.sh

```diff
@@ -64,8 +64,6 @@ run_cmd_expect_failure $XB_BIN $XB_ARGS \
 --prepare --target-dir=$topdir/backup1 --xtrabackup-plugin-dir=${plugin_dir} \
 --component-keyring-file-config=$topdir/xtrabackup_component_keyring_file_invalid.cnf 2>&1 | tee $topdir/pxb.log
 
-grep_count "Component configuration does not have path member" $topdir/pxb.log 1
-
```

Comment:
removed by mistake?

Reply:
No, we no longer print this message at https://github.com/percona/percona-xtrabackup/pull/1226/commits/86941fe7c6a55b509a978fea15ba9c3c4b334a95#diff-d9a0076502e9950744186c9fe95e006525d8f050b1ee5828fcfe7d3e45134a82L217